onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -476,9 +620,7 @@ void addGlobalMethods(py::module& m, const Environment& env) {
 
 #ifdef USE_CUDA
   m.def("set_cuda_device_id", [](const int id) { cuda_device_id = static_cast<OrtDevice::DeviceId>(id); });
```

Comment:
we should mark all these global cuda set_* methods as deprecated. add some comments?

Reply:
comments added.